src/lib/CanvasAnnotationDisplay.js

```diff
@@ -75,6 +79,10 @@ export default class CanvasAnnotationDisplay {
           this.context.globalAlpha = element.attributes['fill-opacity'].nodeValue;
         }
         this.context.fill(p);
+      } else if (this.isSearch) {
```

Comment:
In this case would we have a search annotation that is being represented by an SVG? I guess this is possible?